src/scsi/.gitignore

```diff
@@ -0,0 +1,2 @@
+/target
+target
```

Comment:
is this a duplicate? Are we trying to ignore a file called target as well as the build directory?

Reply:
Not sure what I was thinking there. In any case, this gitignore seems to be an artifact from before I moved the crate into the `vhost-device` repo, and it doesn't do anything now, so I'll remove it.